src/components/header/ProfileThreadTracingMarkerOverview.js

```diff
@@ -15,8 +15,11 @@ export default connect((state, props) => {
   const { threadIndex } = props;
   const selectors = selectorsForThread(threadIndex);
   const selectedThread = getSelectedThreadIndex(state);
+  const intervalMarkers = selectors
+    .getRangeSelectionFilteredTracingMarkers(state)
+    .filter(tm => tm.name !== 'GCMajor');
```

Comment:
I think I had another one in mind but I've forgotten about it now. I think this is fine for now and we can make it more generic once we need it.